packages/core/lib/src/core_html_widget.dart

```diff
@@ -223,10 +222,10 @@ class _RootTsb extends TextStyleBuilder {
     return super.build(context);
   }
 
-  TextStyleHtml builder(TextStyleHtml _, _HtmlWidgetState state) {
-    if (_output != null) return _output;
+  TextStyleHtml builder(TextStyleHtml? _, _HtmlWidgetState? state) {
```

Comment:
```suggestion
  TextStyleHtml builder(TextStyleHtml _, _HtmlWidgetState state) {
```

Reply:
I could not make `TextStyleHtml?` not nullable, that made 637 tests fail ;)